### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by alanzhou-okta**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+Contributing to Okta Open Source Repos
```
Comment (rchild-okta) [inline CONTRIBUTING.md]: Let's add a header style here (i.e. "#")

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## Style
+
+### Git Commit Messages
+
+We use a simplified form of [Atom](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#git-commit-messages) commit convention.
```
Comment (rchild-okta) [inline CONTRIBUTING.md]: Atom's, or "the Atom"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+    <emoji> Short Description of Commit
+    <BLANKLINE>
+    More detailed description of commit
+    <BLANKLINE>
+    Resolves: <Jira # or Issue #>
+
+#### Emoji Categories
+Our categories include:
+  * :package: `:package` when pushing new release
```
Comment (rchild-okta) [inline CONTRIBUTING.md]: pushing **a** new release

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  * :memo: `:memo:` when writing docs
+  * :bug: `:bug:` when fixing a bug
+  * :fire: `:fire:` when removing code or files
+  * :white_check_mark: `:white_check_mark:` when adding tests
+  * :arrow_up: `:arrow_up:` when upgrading dependencies
+  * :arrow_down: `:arrow_down:` when downgrading dependencies
+    
+If you do not see an exact emoji match, use the best matching emoji.
+
+#### Example
```
Comment (rchild-okta) [inline CONTRIBUTING.md]: Also, should we add a link to this in the README?

File: CONTRIBUTING.md (hunk 1 above)
Comment (alanzhou-okta) [inline CONTRIBUTING.md]: whoops, somehow got dropped. fixed

File: CONTRIBUTING.md (hunk 2 above)
Comment (alanzhou-okta) [inline CONTRIBUTING.md]: fixed

File: CONTRIBUTING.md (hunk 3 above)
Comment (alanzhou-okta) [inline CONTRIBUTING.md]: fixed

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
+#Contributing to Okta Open Source Repos
```
Comment (rchild-okta) [inline CONTRIBUTING.md]: nit: space

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+####Template
+
+    <emoji> Short Description of Commit
+    <BLANKLINE>
+    More detailed description of commit
+    <BLANKLINE>
+    Resolves: <Jira # or Issue #>
+
+#### Emoji Categories
+Our categories include:
```
Comment (rchild-okta) [inline CONTRIBUTING.md]: nit, can we order this based on what we think will be the most used? I.e. feature, bug, others

File: README.md [hunk 7]
```diff
@@ -7,6 +7,8 @@ The Okta Sign-In Widget is a Javascript widget that provides a fully featured an
 
 For a high level overview of the widget's features and authentication flows, check out [our developer docs](http://developer.okta.com/code/javascript/okta_sign-in_widget.html).
 
+Contributors should read our [contributing guidelines](https://github.com/okta/okta-signin-widget/blob/master/CONTRIBUTING.md) if they wish to contribute.
```
Comment (rchild-okta) [inline README.md]: You should be able to use a relative link to this I think?

**Pull request merged**